README.md

```diff
@@ -28,3 +28,13 @@ code. The expectation is that `extra-builtins-file` defines a set of
 builtins, but ultimately that's up to the end user.
 
 [config_h_in]: ./nix-plugins-config.h.in
+
+Install
+-------
+
+Add the following to your NixOS configuration:
+```
+nix.extraOptions = ''
+  plugin-files = ${pkgs.nix-plugins}/lib/nix/plugins/libnix-extra-builtins.so
```

Comment:
Can just do `${pkgs.nix-plugins}/lib/nix/plugins` to include any future plugins added here.